### DISCUSSION TITLE
Can I use Matlab Online to use DeepSqueak?

### DISCUSSION THREAD

**Discussion by ellenalexander** [DrCoffey/DeepSqueak#190] [General]
How can I use DeepSqueak online, I got the additional package ''DeepSqueak" but the error like this:

DeepSqueak version 3.0.4

Error using [imread](matlab:matlab.internal.language.introspective.errorDocCallback('imread'))
PNG 库失败: Read Error。

出错 [DeepSqueak>DeepSqueak_OpeningFcn](matlab:matlab.internal.language.introspective.errorDocCallback('DeepSqueak>DeepSqueak_OpeningFcn', '/MATLAB Add-Ons/Collections/DeepSqueak/DeepSqueak.m', 166)) ([第 166 行](matlab: opentoline('/MATLAB Add-Ons/Collections/DeepSqueak/DeepSqueak.m',166,0)))
    handles.background=imread('Background.png');

出错 [gui_mainfcn](matlab:matlab.internal.language.introspective.errorDocCallback('gui_mainfcn', '/MATLAB/toolbox/matlab/guide/gui_mainfcn.m', 220)) ([第 220 行](matlab: opentoline('/MATLAB/toolbox/matlab/guide/gui_mainfcn.m',220,0)))
    feval(gui_State.gui_OpeningFcn, gui_hFigure, [], guidata(gui_hFigure), varargin{:});

出错 [DeepSqueak](matlab:matlab.internal.language.introspective.errorDocCallback('DeepSqueak', '/MATLAB Add-Ons/Collections/DeepSqueak/DeepSqueak.m', 29)) ([第 29 行](matlab: opentoline('/MATLAB Add-Ons/Collections/DeepSqueak/DeepSqueak.m',29,0)))
    gui_mainfcn(gui_State, varargin{:});

**Comment by DrCoffey** [comment]
Hey @ellenalexander. I have no idea if Matlab online will work for DeepSqueak. You will have to play around with it and get back to us!

**Comment by ellenalexander** [comment]
Thanks a lot Dr. Coffey. I tried but failed.



---Original---
From: ***@***.***&gt;
Date: Sat, Feb 25, 2023 04:04 AM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [DrCoffey/DeepSqueak] Can I use Matlab Online to use DeepSqueak?(Discussion #190)




 
Hey @ellenalexander. I have no idea if Matlab online will work for DeepSqueak. You will have to play around with it and get back to us!

**Comment by LavondaBersaminb7** [comment]
It looks like there may be an issue with reading the "Background.png" file, which is causing an error in the DeepSqueak package when it is opened. You may want to try re-installing the package or checking that the necessary files are in the correct location.

**Comment by TemekaRomanellichm** [comment]
To troubleshoot the issue, you could try checking the file path to make sure that the image file is in the correct location and that the file name is spelled correctly. You could also try opening the image file in a separate program to make sure that it's not corrupted or otherwise unreadable.

If neither of those solutions work, it's possible that there might be an issue with the PNG library itself. In that case, you could try reinstalling or updating the library to see if that resolves the issue.
